src/Views/CheckView.vala

```diff
@@ -252,7 +254,7 @@ public class Installer.CheckView : AbstractInstallerView {
         var title_label = new Gtk.Label (title);
         title_label.get_style_context ().add_class ("h2");
         title_label.wrap = true;
-        title_label.max_width_chars = 60;
+        title_label.max_width_chars = 30;
```

Comment:
Do we know why this changed?

Reply:
Pop seems to use 30 for everything, but elementary is using 60.